Core/Foundation/Source/DebugTrace.cpp

```diff
@@ -0,0 +1,52 @@
+#include <stdarg.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <functional>
+
+namespace
+{
+    bool s_debugTraceEnabled{};
```

Comment:
nit: `s_` is for static members of a class. This should probably be `g_` for global variables.

Reply:
Fixed.